src/renderer-process/proxy-window/ui/proxy-dashboard.tsx

```diff
@@ -55,6 +57,7 @@ export const ProxyDashboard = (): JSX.Element => {
     window.desktopApi.getIpAddress();
     window.desktopApi.initFilterRegex();
     window.desktopApi.isRecording();
+    window.desktopApi.port();
```

Comment:
what does this do?

Reply:
Good question. It gets the used port in proxy server (recorder). Maybe it needs a rename.